TOC/Proposal/README.md

```diff
@@ -0,0 +1 @@
+This folder holds proposals that require TOC members to make decisions. The decision-making process follows the voting rules in TiDB community.
```

Comment:
`voting-rules.md` can be inlined into `README.md`

Reply:
OK, thanks.